### PR TITLE
completed react-hot-button-component

### PR DESCRIPTION
**Pull request by m-Cho1**
react-hot-button-component

### CODE REVIEW

File: react-hot-button-component/src/hot-button.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      return (
+        <button onClick={this.handleClick} className={className}>Hot Button</button>
+      );
+    } else {
+      const className = 'hot-btn-white';
+      return (
+        <button onClick={this.handleClick} className={className}>Hot Button</button>
+      );
+    }
+  }
```
Comment (scbowler) [inline react-hot-button-component/src/hot-button.jsx:50]: This is a lot of repetition and would be hard to maintain. You are half way doing it the correct way by saving the class into a variable but the if statements should just determine the class name then at the end have a single return.
```JSX
const clicks = this.state.clicks;
let className = '';

if (clicks < 4) {
  className = 'hot-btn-purple';
} else if (clicks < 7) {
  className = 'hot-btn-light-purple';
} else if (clicks < 10) {
  className = 'hot-btn-red';
} else if (clicks < 13) {
  className = 'hot-btn-orange';
} else if (clicks < 16) {
  className = 'hot-btn-yellow';
} else {
  className = 'hot-btn-white';
}

return (
  <button onClick={this.handleClick} className={className}>
    Hot Button
  </button>
);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but some updates needed. Let me know if you have any questions.

**Comment by m-Cho1** [comment]
Edited conditional statement in render method

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**